app/routes/__frontend/__simpleMdx/about.mdx

```diff
@@ -10,17 +10,17 @@ import { Link } from '@remix-run/react';
 
 ## Who we are
 
-Virtual Coffee is, and always will be, a genuine community of people who value and prioritize supporting one another. We know that growth comes at all levels and that no matter what stage of the developer journey you're on, you can teach and learn.
+Virtual Coffee is, and always will be, a genuine community of people who value and prioritize supporting one another. We know that growth comes at all levels and that you can teach and learn no matter what stage of the developer journey you're on.
 
-Our twice-weekly live coffees with devs at all stages of the journey has grown into an online community that mentors, creates educational content, and provides a safe community at no cost. Since our first coffee in April 2020, we've added a lively slack, a site, Lunch & Learn talks, an Interview Series, a Hacktoberfest Initiative, and Lightning Talks.
+Our twice-weekly live Coffees with devs at all stages of the journey have grown into an online community that mentors, creates educational content, and provides a safe community at no cost. Since our first Coffee in April 2020, we've added a lively Slack, a site, Lunch & Learn talks, an Interview Series, a Hacktoberfest Initiative, and Lightning Talks.
```

Comment:
Suggestion: Since our first Coffee in April 2020, our free, online, tech community has grown to support developers at all stages of the journey. In addition to twice-weekly live Coffees, we've added a site and lively Slack. Our community mentors and creates educational content in a variety of formats including Lunch and Learn talks, a Hacktoberfest Initiative, opportunities to practice public speaking, monthly challenges, Coffee Table Groups, and more.

Reply:
Love it! We definitely need to update that 😁

I want to also mention here that we received the DevRel Awards 2022.
Something like:

Since our first Coffee in April 2020, our free online tech community has grown to support developers at all stages of the journey. In addition to twice-weekly live Coffees, we've added a site and lively Slack. Our community mentors and creates educational content in various formats, including Lunch and Learn talks, a Hacktoberfest Initiative, opportunities to practice public speaking, monthly challenges, Coffee Table Groups, and more. We also won the [DevRel Awards 2022 as the "Most welcoming developer community"](https://youtu.be/uIby7vyYHVg).

Any suggestion?

Reply:
Love it!